src/core/terminology.md

```diff
@@ -141,12 +148,14 @@ There are three coordinate systems for each camera:
             - increasing `x` vector component; increasing `phi`
         - see the cartesian to spherical coordinate calculation in our [source code](https://github.com/pupil-labs/pupil/blob/eb8c2324f3fd558858ce33f3816972d93e02fcc6/pupil_src/shared_modules/pupil_detector_plugins/visualizer_pye3d/utilities.py#L14)
         - an overview of eye model data is in the 'Raw Data Exporter' [documentation](/core/software/pupil-player/#raw-data-exporter)
-    - note that eye model values are reported in opposite directions for the right eye (eye0) as the right eye camera is 
-      physically upside down
-    
-[Reference](https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html).
-You can use the _Camera Intrinsics_ to project a _3d camera location_ to _2d pixel location_, and vice versa.
+    - note that eye model values are reported in opposite directions for the right eye
+        (eye0) as the right eye camera is physically upside down
 
+::: tip
+<v-icon large color="info">info_outline</v-icon>
+If the eye model fails to estimate `phi` or `theta` it will set these two fields and the
```

Comment:
Is it worth mentioning blinks in this case, e.g. "In situations where the eye model fails to estimate `phi` or `theta`, such as during blinks, it will set these two fields and the confidence......"